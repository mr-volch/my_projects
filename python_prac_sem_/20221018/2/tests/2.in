:cos x y cos(x+y)
cos 1 2
:decorate s d "--<<{}>>--".format(s+d)
decorate "ЖЖЖ" "AAA"
cos 0.5 0.7
quit "{} ... {}"
