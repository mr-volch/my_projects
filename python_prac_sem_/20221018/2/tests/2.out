-0.9899924966004454
--<<ЖЖЖAAA>>--
0.3623577544766736
3 ... 6
